Phenotyping thumbnail and thumbnail documentation (#509)
alan-turing-institute/scivision: docs/model_repository_template.md

  - :ref:`config`
  - :ref:`pip`
  <!-- - Model adapter code (TODO: for a later version of scivision)-->

> - :ref:`requirements-for-scivision-website`

Reviewer: Nice, this was exactly what I meant :)

Author: No worries at all. Glad I understood you correctly.

In scivision, once your model(s) have been included in the scivision catalog, pip installability gives users the option to use the `load_pretrained_model` function for easy use of your model code. See the :ref:`api-docs` docs for details.

## 🎆 Model Thumbnail

When viewing the model in [sci.vision](https://sci.vision/#/model-grid) a model thumbnail is required. The thumbnail can be an interesting image explaining the model and that catches the attention of the Scivision user (creativity encouraged!).

- It must be a 256x256 PNG or JPEG file
- The file name should be the name of the model as in the catalog

> Once you have identified a suitable thumbnail please upload it to the [thumbnail models folder](https://github.com/alan-turing-institute/scivision/tree/main/scivision/catalog/data/thumbnails/models).

Reviewer: ```suggestion
Once you have identified a suitable thumbnail, follow the :ref:`how-to-contribute` guide to open a pull request to the Scivision GitHub repo which adds the thumbnail image in the `scivision/catalog/data/thumbnails/models` folder. 
```

In scivision, once your model(s) have been included in the scivision catalog, pip installability gives users the option to use the `load_pretrained_model` function for easy use of your model code. See the :ref:`api-docs` docs for details.

> <<<<<<< HEAD

Reviewer: ```suggestion
```

.. _requirements-for-scivision-website:

## 💻 Requirements for scivision website

.. _model-thumbnail:

### 🎆 Model Thumbnail

> =======

Reviewer: ```suggestion
```

> ## 🎆 Model Thumbnail

Reviewer: ```suggestion
```
removed because it's a duplicate

When viewing the model in [sci.vision](https://sci.vision/#/model-grid) a model thumbnail is required. The thumbnail can be an interesting image explaining the model and that catches the attention of the Scivision user (creativity encouraged!).

- It must be a 256x256 PNG or JPEG file
- The file name should be the name of the model as in the catalog

> <<<<<<< HEAD

Reviewer: ```suggestion
```

> Once you have identified a suitable thumbnail please upload it to the [thumbnail models folder](https://github.com/alan-turing-institute/scivision/tree/main/scivision/catalog/data/thumbnails/models).

Reviewer: ```suggestion
```
Removed as there is the changed version below

> =======

Reviewer: ```suggestion
```
